CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+ All notable changes to this project will be documented in this file.
+ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
+
+  ### [1.0.0] - 2019-02-13
```

Comment:
Added tag to master and updated

Reply:
Headline is still wrong:
- should be h2 (##)
- wrong indention - please remove leading space
- still there's no tag. Please refer to other changelings in our repositories.

Reply:
Made h2 and removed indentation. Will add tag when ready to make a beta for testing like discussed in daily.

Reply:
Please check our changelogs in any of our github repositories. I'd suggest to skip the Diff headline and just make the [1.0.0] to be an actual link.

You can follow the pattern in pwa-extension-kit for example.
And you can already prepare a link to v1.0.0, it will start working at the moment we'll create the v1.0.0 tag.

Reply:
Will follow example